### PR TITLE
Add test for interfaces/webtransport.idl

### PR DESCRIPTION
**Pull request by stephenmcgruer**
Closes https://github.com/web-platform-tests/wpt/pull/25764

### CODE REVIEW

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
@stephenmcgruer have you checked the wpt-chrome-dev-stability failure? Feel free to merge if it's bogus.

**Comment by stephenmcgruer** [comment]
Ooo, it's flaky, fun!

```
 0:42.54 INFO ## Unstable results ##
 0:42.54 INFO |                 Test                 | Subtest |          Results          | Messages |
 0:42.54 INFO |--------------------------------------|---------|---------------------------|----------|
 0:42.54 INFO | `/web-transport/idlharness.any.html` |         | **ERROR: 9/10, OK: 1/10** |          |
```

Ah, there's a race in whether the QuicTransport class realizes there's no connection I think; `ERROR /web-transport/idlharness.any.html - Unhandled rejection: Connection lost.`.

To do properly we need the QUIC server up. Since that's coming soon, I think I'm going to punt on this till then, sorry for pinging you over it too early.

**Comment by stephenmcgruer** [comment]
@Hexcles - is the quic server usable now?

File: web-transport/idlharness.any.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// META: global=window,worker
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
```
Comment (Hexcles) [inline web-transport/idlharness.any.js:3]:
```suggestion
// META: script=/resources/idlharness.js
// META: quic=true
```

**Review by Hexcles — DISMISSED** [review]
It's not enabled yet because of a cert issue.

Regardless, this test should have a label and be skipped if the QUIC server isn't available.

**Comment by stephenmcgruer** [comment]
> Regardless, this test should have a label and be skipped if the QUIC server isn't available.

Ack, but I sorta need the quic server available to develop the test 😆

**Comment by Hexcles** [comment]
Oh I guess you'd need to wait just a bit longer then :)

Besides, I'm not even sure QUIC is enabled on the Chrome side even with `--enable-experimental-web-platform-features`. There were a couple of more flags needed according to https://github.com/web-platform-tests/wpt/pull/22844#issuecomment-611937599

**Comment by stephenmcgruer** [comment]
@Hexcles What's are the plans on making QUIC available for wpt.fyi runs? Looks like these all get marked `SKIP` with the `h2` flag added (note that the test still needs work, but little point if they're just going to be skipped ;)).

**Comment by Hexcles** [comment]
This is blocked by https://github.com/WICG/web-transport/issues/125

**Comment by foolip** [comment]
Hmm, this is still SKIP on wpt.fyi: https://wpt.fyi/results/webtransport/idlharness.any.html?label=pr_head&max-count=1&pr=22852

**Comment by foolip** [comment]
The test now works enough to be useful: https://wpt.fyi/results/webtransport/idlharness.any.html?label=pr_head&max-count=1&pr=22852

**Pull request merged**